src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -16,16 +16,31 @@
 
 using System.Diagnostics.Metrics;
 using System.Reflection;
+using Diagnostics = System.Diagnostics;
 
 namespace OpenTelemetry.Instrumentation.Process;
 
 internal class ProcessMetrics
 {
     internal static readonly AssemblyName AssemblyName = typeof(ProcessMetrics).Assembly.GetName();
     internal static readonly Meter MeterInstance = new(AssemblyName.Name, AssemblyName.Version.ToString());
+    private static readonly Diagnostics.Process CurrentProcess = Diagnostics.Process.GetCurrentProcess();
 
     static ProcessMetrics()
     {
+        // TODO: change to ObservableUpDownCounter
+        MeterInstance.CreateObservableGauge(
+            "process.memory.usage",
+            () => CurrentProcess.WorkingSet64,
```

Comment:
Does `Refresh()` need to be called here? See: https://docs.microsoft.com/dotnet/api/system.diagnostics.process.workingset64#remarks

Reply:
I've added Refresh() before collecting the info; thank you Alan!